.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn.lock
+.idea
```

Comment:
Ideally this is in your global `.gitignore`.

Reply:
Reverted